pkg/controller/issueapi/gen_code.go

```diff
@@ -76,7 +77,14 @@ func (c *Controller) IssueCode(ctx context.Context, vCode *database.Verification
 		stats.Record(ctx, mRealmTokenUsed.M(1))
 	}
 
-	if err := c.CommitCode(ctx, vCode, realm, c.config.GetCollisionRetryCount()); err != nil {
+	if err := c.CommitCode(ctx, vCode, realm, c.config.IssueConfig().CollisionRetryCount); err != nil {
+		if errors.Is(err, database.ErrAlreadyReported) {
+			return &IssueResult{
+				obsResult:   enobs.ResultError("DUPLICATE_USER_REPORT"),
+				HTTPCode:    http.StatusTooManyRequests,
```

Comment:
I feel like StatusTooManyRequests implies rate-limiting? Would conflict be more appropriate?

Reply:
other ideas, just 400? 403?

Reply:
409 would be most appropriate. I think we need to document the response and explain what clients should _do_ instead.